### ISSUE TITLE
MRTK's Radial Prefab is using a font from HoloToolkit

### ISSUE THREAD

**Issue by timGerken** [microsoft/MixedRealityToolkit-Unity#5122]
## Describe the bug

[Radial prefab](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/7b2f03bbf8a2ca360f23da20dd9275f2be277fbd/Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/Radial.prefab#L390) still refers to the old [HoloToolkit selawk font](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/htk_release/Assets/HoloToolkit/UX/Fonts/selawk.ttf.meta) instead of the [MixedRealityToolkit selawk font](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/StandardAssets/Fonts/selawk.ttf.meta).

## To reproduce

1. Open MRTK in Unity
2. Open the Radial prefab
3. Expand the prefab and select the Label component
![image](https://user-images.githubusercontent.com/14828396/60349462-8c19a080-99b9-11e9-96ae-c15994067cd8.png)

Observe the missing font:
![image](https://user-images.githubusercontent.com/14828396/60349421-7a37fd80-99b9-11e9-986e-8ba9fea64d53.png)

## Expected behavior

A proper font.

## Your Setup (please complete the following information)

- Unity Version [e.g. 2018.4.1f1]
- MRTK Version [e.g. v2.0] (both mrtk_release and mrtk_develop branches)

## Target Platform (please complete the following information)

- HoloLens
- HoloLens 2
- WMR immersive
- OpenVR

**Comment by timGerken** [comment]
Note, there are a few other cases of MRTK assets using guids that match those of HoloToolkit .meta files:
```
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit-Examples\StandardShader\Textures\Panel_albedo.tga.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Materials\HolographicButtonContent.mat
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Materials\HolographicButtonContentCage.mat
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Materials\HolographicButtonContentCageProximity.mat
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Materials\HolographicButtonThickContent.mat
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Materials\HolographicButtonThinContent.mat
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit-Examples\UX\Materials\ToolTipPlateEmerald.mat.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.Examples\Demos\Solvers\Prefabs\Snake.prefab
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Fonts\selawk.ttf.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.Examples\Demos\UX\Tooltips\Scenes\TooltipExamples.unity
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.SDK\Features\UX\Interactable\Prefabs\Radial.prefab
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Materials\MRTK_Standard_Lime.mat.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.Examples\Demos\UX\Tooltips\Scenes\TooltipExamples.unity
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit-Examples\Input\Materials\GridCubeWhite.mat.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.Examples\Demos\UX\Tooltips\Scenes\TooltipExamples.unity
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\Boundary\Scripts\SceneContentAdjuster.cs.meta
    C:\src\GitHub\Microsoft\MRTK\Assets\MixedRealityToolkit.Examples\Demos\UX\Tooltips\Scenes\TooltipExamples.unity
```

**Comment by timGerken** [comment]
@Cameron-Micka seem tangentially related to the cool tool you just contributed. Just add the old HoloToolkit to your MRTK project and point the tool at it to see what in the MRTK needs updating.

**Comment by Cameron-Micka** [comment]
@timGerken that’s a great idea! @wiwei we might want to fix all these old references before GA.

**Comment by cre8ivepark** [comment]
Updated interactable prefabs and opened PR. #5421

**Comment by timGerken** [comment]
It looks like there are other issues going on here besides just the errant font usage (some of these were captured in a comment in #5241. Maybe opening a new issue to track the non-font items is the right thing to do, your call. Renaming the existing issue to capture all HTK references in MRTK might also work. 

For instance, the old HoloToolkit's MRTK_Standard_Cyan.mat was renamed to ProgressIndicatorLoadingBarFilled.mat (they share the same guid):
C:\src\GitHub\Microsoft\\**HoloToolkit**\Assets\HoloToolkit\UX\Materials\\**MRTK_Standard_Cyan.mat.meta**
>fileFormatVersion: 2
>guid: **97b73006130d05b4c90034845c44e88d**
>timeCreated: 1435687483
>licenseType: Pro
>NativeFormatImporter:
>  userData:
>  assetBundleName:
>  assetBundleVariant:

C:\src\GitHub\\**cre8ivepark\MixedRealityToolkit-Unity**\Assets\MixedRealityToolkit.SDK\Features\UX\Materials\\**ProgressIndicatorLoadingBarFilled.mat.meta**
>fileFormatVersion: 2
>guid: **97b73006130d05b4c90034845c44e88d**
>timeCreated: 1435687483
>licenseType: Pro
>NativeFormatImporter:
>  userData:
>  assetBundleName:
>  assetBundleVariant:

The renaming bit might be by design, but it certainly makes things difficult when trying to port a HoloToolkit-base project to MixedRealityToolkit.v2.

The full list that I've been able to find looking for HTK references in this MRTK branch follows here:
```
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\Utilities\Scripts\StabilizationPlaneModifier.cs.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Experimental\Features\Utilities\StabilizationPlaneModifier.cs.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Materials\MRTK_Standard_Cyan.mat.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Materials\ProgressIndicatorLoadingBarFilled.mat.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorLoadingBar.prefab
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingObject.prefab
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Materials\MRTK_Standard_White.mat.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Materials\ProgressIndicatorLoadingBarEmpty.mat.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorLoadingBar.prefab
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Progress\ProgressIndicator.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Extensions\SceneTransitionService\DefaultSceneTransitionServiceProfile.asset
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorLoadingBar.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Progress\ProgressIndicatorPrefab.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingObject.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Progress\RotatingOrbs.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingOrbs.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\3DTextSelawik.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Slate\SlateExample.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Text\Scenes\TextPrefabExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\3DTextSelawik.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\3DTextSelawikBold.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Text\Scenes\TextPrefabExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\3DTextSelawikBold.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\3DTextSelawikLight.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Text\Scenes\TextPrefabExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\3DTextSelawikLight.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\3DTextSelawikSemibold.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Text\Scenes\TextPrefabExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\3DTextSelawikSemibold.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\3DTextSelawikSemilight.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\StandardShader\Scenes\ClippingExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.Examples\Demos\UX\Text\Scenes\TextPrefabExamples.unity
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\3DTextSelawikSemilight.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\UITextSelawik.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\UITextSelawik.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\UITextSelawikBold.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\UITextSelawikBold.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\UITextSelawikLight.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\UITextSelawikLight.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\UITextSelawikSemibold.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\UITextSelawikSemibold.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Text\UITextSelawikSemilight.prefab.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\StandardAssets\Prefabs\Text\UITextSelawikSemilight.prefab.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Scripts\Progress\ObjectDisplayScript.cs.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingObject.prefab
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Scripts\ProgressIndicators\ProgressIndicatorObjectDisplay.cs.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Scripts\Progress\ProgressIndicator.cs.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorLoadingBar.prefab
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Scripts\ProgressIndicators\ProgressIndicatorLoadingBar.cs.meta
C:\src\GitHub\Microsoft\HoloToolkit\Assets\HoloToolkit\UX\Scripts\Progress\ProgressIndicatorOrbsRotator.cs.meta
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingOrbs.prefab
    C:\src\GitHub\cre8ivepark\MixedRealityToolkit-Unity\Assets\MixedRealityToolkit.SDK\Features\UX\Scripts\ProgressIndicators\ProgressIndicatorOrbsRotator.cs.meta

**Comment by keveleigh** [comment]
I think this one's good after #5421 and #5484, but please reopen if not!

**Comment by timGerken** [comment]
@keveleigh 
Looks like there is just one more:
`MRTK\Assets\MixedRealityToolkit.Extensions\SceneTransitionService\DefaultSceneTransitionServiceProfile.asset` references `HoloToolkit\Assets\HoloToolkit\UX\Prefabs\Progress\ProgressIndicator.prefab.meta` as its `defaultProgressIndicatorPrefab`.

**Comment by keveleigh** [comment]
@timGerken Oops, looks like I accidentally filtered out .asset files when I replaced them in #5484. Fix here: #5567
